packages/core/src/spec/lib/TwaManifestSpec.ts

```diff
@@ -323,4 +323,115 @@ describe('TwaManifest', () => {
       expect(asDisplayMode('')).toBeNull();
     });
   });
+  describe('#merge', () => {
+    it('Validates that the merge is done correctly in case which' +
+        ' there are no files to ignore', async () => {
+      const webManifest: WebManifestJson = {
+        'display': 'fullscreen',
+        'name': 'name',
+        'short_name': 'different_name',
+        'start_url': 'https://name.github.io/',
+        'icons': [{
+          'src': 'https://image.png',
+          'sizes': '512x512',
+          'purpose': 'any',
+        },
+        ],
+      };
+      const twaManifest = new TwaManifest({
+        'packageId': 'id',
+        'host': 'host',
+        'name': 'name',
+        'launcherName': 'name',
+        'display': 'standalone',
+        'themeColor': '#FFFFFF',
+        'navigationColor': '#000000',
+        'navigationColorDark': '#000000',
+        'navigationDividerColor': '#000000',
+        'navigationDividerColorDark': '#000000',
+        'backgroundColor': '#FFFFFF',
+        'enableNotifications': false,
+        'startUrl': '/',
```

Comment:
Can you add a comment somewhere that the start_urls are different, but since they both resolve the same relative to the host url, nothing changes.